src/Arcus.Messaging.Abstractions/Extensions/MessageHandlerCollectionExtensions.cs

```diff
@@ -154,10 +157,11 @@ public static MessageHandlerCollection WithFallbackMessageHandler<TMessageHandle
         /// <param name="collection">The collection to add the fallback message handler to.</param>
         /// <param name="createImplementation">The function to create the fallback message handler.</param>
         /// <exception cref="ArgumentNullException">Thrown when the <paramref name="collection"/> or the <paramref name="createImplementation"/> is <c>null</c>.</exception>
+        [Obsolete("Will be removed in v3.0 as only concrete implementations of message handling will be supported from now on")]
```

Comment:
So this means the complete concept of `FallbackMessageHandler`s will be removed ?

Reply:
Ah, I see.....   Now I understand.
We have 'AzureServiceBusFallbackMessageHandler', and we have this 'general' ones that are actually never used atm.
Should someone ever wanted to create support for let's say RabbitMQ, these 'concepts' would need to be created specifically for RabbitMQ itself.

Reply:
Yes, they are used implicitly in both the Service bus and EventHubs routing, but only there because we supported general message routing in the past. And since Service bus has a special case of message routing and we removed the EventHubs routing, there is no need to have a fully implemented general message routing available.

Reply:
Can we resolve this @fgheysels ?